<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局背景 */
QWidget#Settings {
    background-color: rgba(0, 0, 0, 180);
}

/* 面板 */
QFrame#mainFrame {
    background-color: rgba(30, 30, 30, 230);
    border: 1px solid rgba(80, 80, 80, 200);
    border-radius: 12px;
}

/* 标题 */
QLabel#titleLabel {
    color: white;
    font-size: 24px;
    font-weight: bold;
    padding: 10px;
    background-color: rgba(50, 50, 50, 220);
    border-bottom: 1px solid rgba(80, 80, 80, 200);
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
}

/* 普通标签 */
QLabel {
    color: #cccccc;
    font-size: 16px;
    font-weight: bold;
}

/* 滑块 */
QSlider::groove:horizontal {
    border: 1px solid #bbb;
    background: #444;
    height: 8px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eee, stop:1 #ccc);
    border: 1px solid #777;
    width: 18px;
    margin: -2px 0;
    border-radius: 9px;
}

QSlider::sub-page:horizontal {
    background: #4CAF50;
    border: 1px solid #4CAF50;
    height: 10px;
    border-radius: 4px;
}

/* 按钮 */
QPushButton {
    background-color: rgba(60, 60, 60, 200);
    border: 1px solid rgba(100, 100, 100, 150);
    border-radius: 8px;
    color: white;
    font-size: 16px;
    font-weight: bold;
    padding: 12px;
    min-width: 120px;
}

QPushButton:hover {
    background-color: rgba(85, 85, 85, 220);
    border-color: rgba(150, 150, 150, 200);
}

QPushButton:pressed {
    background-color: rgba(40, 40, 40, 240);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QFrame" name="mainFrame">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>声音设置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="musicLabel">
          <property name="text">
           <string>音乐音量</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="musicSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="musicValueLabel">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sfxLabel">
          <property name="text">
           <string>音效音量</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sfxSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="sfxValueLabel">
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
